vllm/model_executor/models/adapters.py

```diff
@@ -42,13 +42,14 @@ def _create_pooling_model_cls(
     default_softmax: bool,
 ) -> _T:
     # Lazy import
-    from vllm.model_executor.layers.pooler import Pooler, PoolerOutput
-    from vllm.model_executor.pooling_metadata import PoolingMetadata
+    from vllm.model_executor.layers.pooler import Pooler
 
     from .utils import AutoWeightsLoader, WeightsMapper
 
     class ModelForPooling(orig_cls, VllmModelForPooling):
 
+        is_pooling_model = True
+
```

Comment:
Isn't this achieved already by deriving `VllmModelForPooling`?

In the doc we also say that all pooling models implement `VllmModelForPooling` but not all do. Could this be cause of confusion?